include/common_package/topic_names.hpp

```diff
@@ -3,7 +3,7 @@
 namespace common_lib {
```

Comment:
```suggestion
namespace common_lib::topic_names {
```

Reply:
That's my fault, but we can change it in this PR anyway

Reply:
Only one feature per PR. REJECTED!🧐

Reply:
Fixed in 143f8bc